Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
 Shutdown
 #######################
 
-Under construction.
+Stopping Scheduler
+^^^^^^^^^^^^^^^^^^
+
+The Scheduler automatically adds new targets to the queue until the start of nautical morning twilight.
+If there are no targets, the Scheduler will stop adding new ones.
+If you need to stop the Scheduler manually, you can use the ``auxtel/scheduler/stop.py`` script from ScriptQueue without any additional configurations.
+
+See more detailed descriptions on `Scheduler <https://obs-ops.lsst.io/Nighttime-Operations/Scheduler/night-time-operations/index.html>`__.
+
```

Comment:
May be this part would go under Prerequisites? I mean what is needed before in order to start the shutdown and park of auxtel.

Reply:
I don't agree that every page needs to have the same format, such as Pre-reqs, Procedure, Post-conditions. I think if you describe that all in the introduction, there is no need to make different sections.

Reply:
Thanks for the clarification, Alysha. The page did not have Prerequisites, Procedure, Post-conditions, etc whereas the Parking Auxtel has that. I just keep it as it is but I will add it if necessary.

Reply:
I don't agree with Alysha on that if we want to keep things organized and structure for our own comprehension and friendly understanding of following procedures specially if we are building and the pages and putting the inicial information for the future of operations.

Reply:
Whatever you think is appropriate. I'm not worried about it.